modules/electromagnetics/test/tests/auxkernels/azimuthal_Faradays_law/scalar_azim_magnetic_time_deriv_convergence.py

```diff
@@ -0,0 +1,66 @@
+# Script to run spatial convergence study on microwave_heating.i
```

Comment:
this this script
```suggestion
# Script to run spatial convergence study
```

Reply:
Edited it to reference the correct input file.